playcanvas-devtools/readme.md

```diff
@@ -0,0 +1,69 @@
+![Preview](images/preview-20200715-152807.jpg)
+# How to add to a PlayCanvas App
```

Comment:
Shouldn't all this be integrated into the top level README?

Reply:
I was 50/50 on this at the time but have moved it to top level and updated with the new instructions on adding it the app.